types/src/connectors/utils.ts

```diff
@@ -0,0 +1 @@
+export const MAX_FILE_SIZE = 128 * 1024 * 1024; // 200 MB in bytes.
```

Comment:
Can we make this variable name more explicit?
I would go for `MAX_FILE_SIZE_TO_DOWNLOAD`

Reply:
Applied!